examples/datasets_converters/convert_inshop.py

```diff
@@ -68,10 +68,10 @@ def build_inshop_df(dataset_root: Path, no_bboxes: bool) -> pd.DataFrame:
 
     cols_to_pick = ["label", "path", "split", "is_query", "is_gallery", "category"]
 
-    if no_bboxes:
-        df["path"] = df["image_name"].apply(lambda x: Path(dataset_root) / x.replace("img/", "img_highres/"))
+    df["path"] = df["image_name"].apply(lambda x: Path(dataset_root) / x.replace("img/", "img_highres/"))
 
-    else:
+    need_bboxes = not no_bboxes
```

Comment:
Do you really need to introduce a new variable just to use it in one line below?

Reply:
for better readability

Reply:
Can't agree with you that it is more readable. You have `no_bboxes` argument of the function that is used just once to create a new variable `need_bboxes`. As I see it, it is better to rename the argument `no_bboxes` to `need_bboxes` in the function signature, and possibly in the script arguments ([line 123](https://github.com/OML-Team/open-metric-learning/blob/556f692f83a669c57bbf60f2efccf55f350abcc5/examples/datasets_converters/convert_inshop.py#L123))